### PR TITLE
moved rules to their own files and updated a flow rule

### PR DESCRIPTION
**Pull request by TomPridham**
i just broke these out into separate files to make them easier to find. i can move any you disagree with.

I also added an option to a flow rule

### CODE REVIEW

File: flow.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export const flowRules = {
+  'flowtype-errors/show-errors': 2,
+  'flowtype/generic-spacing': 0,
+  'flowtype/no-types-missing-file-annotation': 2,
+  'flowtype/no-weak-types': 2,
+  'flowtype/require-parameter-type': [
+    2,
+    { excludeArrowFunctions: 'expressionsOnly' },
```
Comment (TomPridham) [inline flow.js]: i added this

**Comment by lukeschunk** [comment]
one file to rule them all

File: flow.js (hunk 1 above)
Comment (zacanger) [reply flow.js]: I kinda disagree with this addition. What's your reasoning?

**Comment by lukeschunk** [comment]
actually i'm good with it broken out 👍

File: flow.js (hunk 1 above)
Comment (TomPridham) [reply flow.js]: it pairs with the `require-return-type` rule so that you don't have to do `(x: string) => x`. you can just do `(x) => x`

Comment (zacanger) [reply flow.js]: Right, but what's the reasoning behind making that looser? In general we keep our Flow rules pretty strict.

Comment (TomPridham) [reply flow.js]: im hoping it will make turning this rule on in seller less painful. there are 775 errors with it just set to `2` 😭 vs 208 with it on

Comment (zacanger) [reply flow.js]: I would override that in that app's Eslintrc, then.

**Pull request merged**